libs/core/src/lib/dialog/dialog-close-button/dialog-close-button.component.ts

```diff
@@ -15,6 +15,7 @@ import { Component, Input } from '@angular/core';
         '[class.fd-button]': 'true',
         '[class.fd-button--compact]': '!mobile',
         '[class.fd-button--transparent]': 'true',
+        'title': 'close'
```

Comment:
what about i18n support for this static text?

Reply:
modified

Reply:
as mentioned above, `title` should not be hardcoded here if you cannot also get `i18n-title` here.